components/Users/UserModal/userModal.tsx

```diff
@@ -58,65 +126,177 @@ const UserModal: React.FunctionComponent<UserModalProps> = ({
+                            className={styles["inLineEditIcon"]}
+                            type={"edit"}
+                          />
+                        </div>
+                        <div className={styles["editSection"]}>
+                          <input
+                            className={styles["editTitle"]}
+                            defaultValue={
+                              currUser ? currUser.email : user?.email
+                            }
+                            onChange={(e) => setEmail(e.target.value)}
+                          />
+                          <Icon
+                            className={styles["inLineEditIcon"]}
+                            type={"edit"}
+                          />
+                        </div>
+                        <div className={styles["editSection"]}>
+                          <input
+                            className={styles["editTitle"]}
+                            defaultValue={
+                              currUser
+                                ? currUser.phone_number
+                                : user?.phone_number
+                            }
+                            onChange={(e) => setPhoneNumber(e.target.value)}
+                          />
+                          <Icon
+                            className={styles["inLineEditIcon"]}
+                            type={"edit"}
+                          />
+                        </div>
+                        <div className={styles["infoSpacing"]}>
+                          <input
+                            className={styles["password"]}
+                            value="password"
+                            type="text"
+                          />
+                          <div className={styles["passwordButton"]}>
+                            <Icon
+                              className={styles["resetIcon"]}
+                              type={"hidepassword"}
+                            />
+                            <a className={styles["resetText"]}>
+                              Reset Password
+                            </a>
+                          </div>
+                        </div>
+                      </div>
+                    ) : (
+                      <div>
+                        <div className={styles["infoSpacing"]}>
+                          {currUser ? currUser.role : user?.role}
+                        </div>
+                        <div className={styles["infoSpacing"]}>
+                          {currUser ? currUser.email : user?.email}
+                        </div>
+                        <div className={styles["infoSpacing"]}>
+                          {currUser
+                            ? currUser.phone_number
+                            : user?.phone_number}
+                        </div>
+                        <div className={styles["infoSpacing"]}>
+                          <input
```

Comment:
I don't think we need to display the password anymore if we're just gonna have admins reset them. Thoughts @hukellyy?